tests/test_command_arf_to_graph.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import os
+import pexpect
+import pytest
+from readchar import key
+
+import tests.any_test_help
+
+
+@pytest.mark.usefixtures("remove_generated_reports_in_root")
+def test_command_arf_to_graph():
+    out = subprocess.check_output(['python3',
+                                   '-m',
+                                   'oval_graph.command_line',
+                                   'arf-to-graph',
+                                   '--off-web-browser',
+                                   'tests/test_data/ssg-fedora-ds-arf.xml',
+                                   'xccdf_org.ssgproject.content_rule_package_abrt_removed'])
+    tests.any_test_help.compare_results_html(
+        out.decode('utf-8').split('\n')[-2])
+
+
+def test_command_arf_to_graph_with_out_parameter():
+    src = tests.any_test_help.get_random_dir_in_tmp()
+    subprocess.call(
+        ('python3 -m oval_graph.command_line arf-to-graph --off-web-browser ' +
+         '--out ' + src +
+         ' tests/test_data/ssg-fedora-ds-arf.xml xccdf_org.ssgproject.' +
+         'content_rule_package_abrt_removed'), shell=True)
```

Comment:
This is not correct, because you pass a single string, and then you have to specify `shell=True`, so the string is decomposed into the command and its arguments. You can fix this easily, as `subprocess.call` accepts a list/tuple of command components.